src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/ItemsResultsList/ItemsResultsList.js

```diff
@@ -155,7 +160,11 @@ export default class ItemsResultsList extends Component {
       },
     ];
 
-    return !_isEmpty(results.hits) ? (
+    return isLoading && executedSearch ? (
+      <Loader isLoading={isLoading}>
+        <this.renderAll />
```

Comment:
I am not sure this `renderAll` is correct, I cannot find it. What was the expected behaviour?

Reply:
No it's not correct, thanks for noticing. I probably forget to delete it from a copy paste, the expected behaviour is that if it is executing the searching query avoid to display the `Found no items...` and display the `loader` even if it only lasts some miliseconds before displaying the result list (if there are elements)